src/pgrn-variables.c

```diff
@@ -295,6 +316,25 @@ PGrnInitializeVariables(void)
 							 PGrnLogLevelAssign,
 							 NULL);
 
+#if GRN_VERSION_OR_LATER(14, 0, 7)
+	DefineCustomIntVariable(
+		"pgroonga.log_rotate_threshold_size",
+		"PGroonga log rotation threshold.",
+		"Specifies threshold for log rotation. "
+		"Log file is rotated when log file size is larger "
+		"than or equals to the threshold (default: 0; disabled). "
+		"Unit is bytes.",
```

Comment:
Do we need white spaces for this description?
```suggestion
		"pgroonga.log_rotate_threshold_size",
		"PGroonga log rotation threshold.",
		"Specifies threshold for log rotation."
		"Log file is rotated when log file size is larger "
		"than or equals to the threshold (default: 0; disabled)."
		"Unit is bytes.",
```

Reply:
これ、実は連結されて1行になるんです。
ので、`.` のあとにもスペースを入れています。